alias-repository.js

```diff
@@ -438,7 +438,8 @@ class AliasRepository extends Array {
         return canonical
             .split(separator)
             .map((tk) => gHelper.toTitleCase(tk))
-            .join(' ');
+            .join(' ')
+            .replace(/ \(.*\)/, '');
```

Comment:
Parentheses can be used to avoid clashing aliases; they will be cut out of being formatted.